doc/src/vpr/command_line_usage.rst

```diff
@@ -626,6 +626,12 @@ For people not working on CAD, you can probably leave all the options to their d
 
     **Default:** ``2``
 
+.. option:: --write_block_usage <file>
+
+    Writes out to the file under path <file> block usage summary in machine
```

Comment:
One tweak: should say this is the cluster-level block usage summary (here and ideally at the top of the file printed out).

Reply:
Fixed